src/Orc.DataAccess/Database/Extensions/DbConnectionStringExtensions.cs

```diff
@@ -1,12 +1,16 @@
 ﻿namespace Orc.DataAccess.Database
 {
+    using System;
     using Catel;
 
     public static class DbConnectionStringExtensions
     {
-        public static DbConnectionStringProperty TryGetProperty(this DbConnectionString connectionString, string propertyName)
+        public static DbConnectionStringProperty? TryGetProperty(this DbConnectionString connectionString, string propertyName)
```

Comment:
TryGetProperty should not return a value but a bool whether it succeeded. Otherwise it should be GetValue(string propertyName)